### PR TITLE
LPD-33808 InfoBox for shippingMethods

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/LPD-33808

### CODE REVIEW

File: modules/apps/commerce/commerce-order-content-web/src/main/resources/META-INF/resources/js/info_box/modal/InfoBoxModalShippingMethodInput.js [hunk 1]
```diff
@@ -36,6 +44,19 @@ const InfoBoxModalShippingMethodInput = ({
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, []);
 
+	useEffect(() => {
+		setSelectedShippingMethod(
+			shippingMethods.find(
+				(shippingMethod) =>
+					shippingMethod.engineKey === shippingMethodEngine
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-order-content-web/src/main/resources/META-INF/resources/js/info_box/modal/InfoBoxModalShippingMethodInput.js:52]: shippingMethodEngine(Key?)

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:playwright

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**